app/src/test/java/nerd/tuxmobil/fahrplan/congress/utils/ConferenceTimeFrameTest.kt

```diff
@@ -0,0 +1,107 @@
+package nerd.tuxmobil.fahrplan.congress.utils
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Assert.fail
+import org.junit.Before
+import org.junit.Test
+
+class ConferenceTimeFrameTest {
+
+    private companion object {
+        // 2015-12-27T00:00:00+0100, in seconds: 1451170800000
+        const val FIRST_DAY_START_TIME = 1451170800000L
+
+        // 2015-12-31T00:00:00+0100, in seconds: 1451516400000
+        const val LAST_DAY_END_TIME = 1451516400000L
+    }
+
+    private lateinit var conference: ConferenceTimeFrame
+
+    @Before
+    fun setUp() {
+        conference = ConferenceTimeFrame(FIRST_DAY_START_TIME, LAST_DAY_END_TIME)
+    }
+
+    @Test
+    fun isValidWithFirstDayThenLastDay() {
```

Comment:
as you were making the test names more readable in the other tests, let's do it here too?

Reply:
:x: Your comment seems to be based on an outdated version of the `ConferenceTimeFrameTest.kt` file.